groups/alteryx-sandbox/data.tf

```diff
@@ -55,3 +55,11 @@ data "vault_generic_secret" "security_s3_buckets" {
 data "aws_acm_certificate" "acm_cert" {
   domain = var.domain_name
 }
+
+data "vault_generic_secret" "aws_s3_bucket" {
+  path = "applications/${var.aws_account}-${var.aws_region}/${var.application}/s3_buckets/"
+}
+
+data "aws_s3_bucket" "resources" {
+  bucket = "${var.aws_account}.${var.aws_region}.resources.ch.gov.uk"
+}
```

Comment:
`git` expects a newline to be present at the end of the file and will a warning will be displayed in local diffs and here on GitHub. Check your text editor/IDE for an option to include a final trailing newline character.

Reply:
Have now enable this setting.